app/Http/Controllers/UsersController.php

```diff
@@ -130,10 +130,8 @@ public function postCreate(Request $request)
             }
             return redirect::route('users')->with('success', trans('admin/users/message.success.create'));
         } else {
-            redirect()->back()->withInput()->withInput()->withErrors($user->getErrors())->withErrors($settings->getErrors());
```

Comment:
This isn't a copy+paste error, it's because we're doing user model validation and settings model validation